crest/Assets/Crest/Crest-Examples/BoatDev/Scripts/BoatAlignNormal.cs

```diff
@@ -120,6 +122,11 @@ void FixedUpdate()
         var buoyancy = -Physics.gravity.normalized * _buoyancyCoeff * bottomDepth * bottomDepth * bottomDepth;
         _rb.AddForce(buoyancy, ForceMode.Acceleration);
 
+        // Approximate hydrodynamics of sliding along water
+        if (_accelerateDownhill > 0f)
+        {
+            _rb.AddForce(new Vector3(normal.x, 0f, normal.z) * -Physics.gravity.y, ForceMode.Acceleration);
```

Comment:
Is this suppose to use the value of _accelerateDownhill? Otherwise, it could be a boolean.

Reply:
It was actually supposed to be a multiplier! Fixed in the latest. Does it make more sense now?

Reply:
It does. You missed this one again though. Sorry, I should have marked both of them.